openedx/core/lib/gating/services.py

```diff
@@ -0,0 +1,47 @@
+"""
+A wrapper class around requested methods exposed in api.py
+"""
+import types
+
+from . import api as gating_api
+
+
+class GatingService(object):  # pylint: disable=too-few-public-methods
+    """
+    An xBlock service for xBlocks to talk to the Gating api.
```

Comment:
Nit: It's XBlock, not xBlock (we are horribly inconsistent ourselves on this, but that's the official spelling).

Reply:
ack